playbooks/roles/ocp-config/tasks/restart-config.yaml

```diff
@@ -0,0 +1,49 @@
+---
+# Add restart always configurations to the services
+- name: Create squid dropin directory
+  file:
+    path: /etc/systemd/system/squid.service.d
+    state: directory
+    mode: 0755
+  become: true  
+  when: setup_squid_proxy
+    
+- name: Create conf file in the squid dropin directory
+  file:
+    path: /etc/systemd/system/squid.service.d/restart.conf
+    state: touch
+  become: true
+  when: setup_squid_proxy
+    
+- name: Add restart always Service configuration for squid service
+  shell: |
+    named_systemd_dir=/etc/systemd/system/squid.service.d
+    echo "[Service]" | sudo tee $named_systemd_dir/restart.conf
+    echo "Restart=always" | sudo tee -a $named_systemd_dir/restart.conf    
+  notify:
+    - restart squid.d
+  when: setup_squid_proxy
+
+- name: Create dropin directory for services
+  file:
+    path: /etc/systemd/system/{{item}}.service.d
+    state: directory
+    mode: 0755
+  become: true
+  loop: "{{ services }}"
+    
+- name: Create conf file in the services dropin directory
+  file:
+    path: /etc/systemd/system/{{item}}.service.d/restart.conf
+    state: touch
+  become: true
+  loop: "{{ services }}"
+    
+- name: Add restart always Service configuration for services
+  shell: |
+    named_systemd_dir=/etc/systemd/system/{{item}}.service.d
+    echo "[Service]" | sudo tee $named_systemd_dir/restart.conf
+    echo "Restart=always" | sudo tee -a $named_systemd_dir/restart.conf
+  loop: "{{ services }}"    
+  notify:
```

Comment:
Can't the two tasks be combined into a single copy task ? 
Or how about using file module to create the restart.conf content?

Reply:
Sure @bpradipt I will take a look it. Thanks